tests/test_fabricpath_global.rb

```diff
@@ -42,213 +40,203 @@ def no_feature_fabricpath
     config('no feature-set fabricpath')
   end
 
-  def n5k6k_platforms?
-    /N[56]K/ =~ node.product_id
-  end
-
   # TESTS
 
-  def test_global_collection_empty
-    globals = FabricpathGlobal.globals
-    assert_equal(true, globals.empty?,
-                 'Globals should be empty for this test')
-  end
+  def test_create_destroy
+    assert_empty(FabricpathGlobal.globals)
 
-  def test_global_create
-    @global = FabricpathGlobal.new('default')
-    assert_equal(true, @global.name == 'default',
-                 "Global name not set correctly #{@global.name}")
-    assert_equal(:enabled, FabricpathGlobal.fabricpath_feature,
-                 'Fabricpath feature should have been enabled')
-    refute(FabricpathGlobal.globals.empty?,
-           'Globals should not be empty after create')
-  end
+    # create
+    fg = FabricpathGlobal.new('default')
+    assert_equal('default', fg.name)
+    assert_equal(:enabled, FabricpathGlobal.fabricpath_feature)
+    refute_empty(FabricpathGlobal.globals)
 
-  def test_global_destroy
-    # create and test again
-    test_global_create
-    @global.destroy
-    # now it should be wiped out
-    test_global_collection_empty
+    # destroy
+    fg.destroy
+    assert_empty(FabricpathGlobal.globals)
   end
 
   def test_aggregate_multicast_routes
-    return if n5k6k_platforms?
-    @global = FabricpathGlobal.new('default')
-    @global.aggregate_multicast_routes = true
-    assert(@global.aggregate_multicast_routes,
-           'Aggregate multicast routes not set')
-    @global.aggregate_multicast_routes = false
-    refute(@global.aggregate_multicast_routes,
-           'Aggregate multicast routes not reset')
+    fg = FabricpathGlobal.new('default')
+    if validate_property_excluded?('fabricpath', 'aggregate_multicast_routes')
+      assert_raises(Cisco::UnsupportedError) do
+        fg.aggregate_multicast_routes = true
+      end
+      return
+    end
+
+    fg = FabricpathGlobal.new('default')
+    fg.aggregate_multicast_routes = true
+    assert(fg.aggregate_multicast_routes,
+           'aggregate_multicast_routes: Expected: true')
```

Comment:
Nah, it's just syntactic sugar.
When `assert/refute` fail they don't print out a pretty `Expected: foo / Actual: bar` failure like `assert_equal` does. 
Instead you get `Failed assertion, no message given.`